asserts/database_test.go

```diff
@@ -687,14 +689,33 @@ ZF5jSvRDLgI=`
 
 	cfg := &asserts.DatabaseConfig{
 		KeypairManager: asserts.NewMemoryKeypairManager(),
-		Trusted:        []asserts.Assertion{tKey.(*asserts.AccountKey)},
+		Backstore:      asserts.NewMemoryBackstore(),
+		Trusted: []asserts.Assertion{
+			asserts.BootstrapAccountForTest("can0nical"),
+			tKey.(*asserts.AccountKey),
+			// to verify now required devel1 Account assertion
+			asserts.BootstrapAccountKeyForTest("can0nical", testPrivKey1.PublicKey()),
+		},
 	}
 	db, err := asserts.OpenDatabase(cfg)
 	c.Assert(err, IsNil)
 
 	a, err := asserts.Decode([]byte(testAccKey))
 	c.Assert(err, IsNil)
 
+	// prereq Account assertion
+	headers := map[string]string{
+		"authority-id": "can0nical",
+		"account-id":   "developer1",
+		"display-name": "Dev 1",
+		"validation":   "unproven",
+		"timestamp":    "2015-01-01T14:00:00Z",
+	}
+	acct, err := asserts.AssembleAndSignInTest(asserts.AccountType, headers, nil, testPrivKey1)
+	c.Assert(err, IsNil)
```

Comment:
really want to limit the usage of assertstest in database_test iself otherwise is getting a bit too circular